etl/steps/data/garden/tourism/2023-05-05/unwto.meta.yml

```diff
@@ -0,0 +1,330 @@
+dataset:
+  title: UNWTO Tourism Statistics Database
+  description: |-
```

Comment:
This description is repeated on three places - if you find yourself tediously copy&pasting text then let me know, that would mean we have a shitty tooling and should improve it.

Metadata is propagated from previous steps if you set `default_metadata=ds_meadow.metadata` like this
```python
ds_garden = create_dataset(dest_dir, tables=[tb_garden], default_metadata=ds_meadow.metadata)
```
(fix it [here](https://github.com/owid/etl/pull/1145/files#diff-609662235bdca717db84a8ba5f3a5a0bd4957c36997603bb588b58933ce521ceR175) and [here](https://github.com/owid/etl/pull/1145/files#diff-c5773e788fa3d5fcbcf3b725c0607379d9e06a005f452309b0161f42692bbd99R54))

With that, you won't need any of title, description, licenses or sources (you'll actually end up with only `tables:` section.). Setting them here in the YAML file would overwrite default from snapshot / meadow. 

Check your other datasets too.

Reply:
I was getting errors because of new variables I was creating in garden which I guess weren't consistent with the meadow ones ? so this was my fix haha :) I will give it a go again now

Reply:
ah wait sorry this works. so it's this that was failing when it was set like = meadow -->  tb_garden = Table(df, short_name="oil_spills")